src/tools.h

```diff
@@ -156,6 +156,13 @@ class CopyItem : public zim::writer::Item         //Article class that will be p
     {
        return std::unique_ptr<zim::writer::ContentProvider>(new ItemProvider(item));
     }
+
+    zim::writer::Hints getHints() const {
+      auto mimetype = item.getMimetype();
+      auto front_article = ( mimetype.find("text/html") != std::string::npos
+                          && mimetype.find("raw=true") == std::string::npos);
```

Comment:
Convert this piece of code to a free helper function `bool is_front_article(const zim::Item& item)` and reuse it in `PatchItem`.

Reply:
Done. Using name `guess_is_front_article` because it is a guess. mimetype is not a real criteria, but we don't have best.